packages/in_app_purchase/in_app_purchase_android/pubspec.yaml

```diff
@@ -21,6 +21,7 @@ dependencies:
   flutter:
     sdk: flutter
 
+  json_annotation: ^4.0.1
```

Comment:
Are the changes in in_app_purchase_android and in_app_purchase_ios here intentional?

Reply:
Yes, they have been solved with PR #3879 which wasn't merge at that time. Now that it is merged and I rebased the changes have been removed from this PR.